addons/mrp/migrations/13.0.2.0/post-migration.py

```diff
@@ -272,6 +272,40 @@ def fill_planned_datetime(env):
     """)
 
 
+def fill_stock_move_bom_line_id(env):
+    """v13 added a new functionality to this bom_line_id field.
+    A part from the usual value for raw moves in mrp productions,
+    now this field has a value for delivery moves generated from a kit.
+    See _prepare_phantom_move_values method, that has bom_line_id field."""
+    openupgrade.logged_query(
+        env.cr, """
+    UPDATE stock_move sm0
+    SET bom_line_id = mbl0.id
+    FROM (
+        SELECT moves.id, moves.product_id,
+            (array_agg(moves.bom_id))[1] as bom_id

```

Comment:
Why not min or max

Reply:
Because I'm not interested in the min or max. I select the first in the selected order, that's why I put `ORDER BY sm.id, mb.sequence, mb.product_id`.

Reply:
![Selection_129](https://user-images.githubusercontent.com/25005517/120336991-32cbf180-c2f3-11eb-8ec4-d36900b95559.png)